index.html

```diff
@@ -0,0 +1,236 @@
+                     websites.</p>
+                </div>
+              </div>
+
+              <div class="card" style="width: 20rem; padding: 12px; margin: 0 auto;">
+                <div class="card-img-top chicken">
+                  <img src="./images/connect.svg" alt="">
+                </div>
+                <div class="card-body">
+                  <h6>Connect</h6>
+                  <p>You’re not alone. Learn and get help
+                     from our friendly community of
+                     beginner and experienced 
+                     developers.</p>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="topics-section" style="background-color: #f1f1f1; padding-bottom: 36px;">
+
+        <header>
+          <h5 style="padding: 32px 0;">Learn everything you need to know to become a web developer</h5>
+        </header>
+
+        <div class="container">
+          <div class="row">
+            <div class="col-xl-8 offset-xl-2">
+
+              <div class="cards cards-topics">
+
+                <div class="card topics" style="width: 13rem;">
+                  <div class="card-body chicken">
+                    <img src="./images/html.svg" alt="">
+                    <h6>HTML + CSS</h6>
+                  </div>
+                </div>
+
+                <div class="card topics" style="width: 13rem;">
+                  <div class="card-body chicken">
+                    <img src="./images/js.svg" alt="">
+                    <h6>JAVASCRIPT</h6>
+                  </div>
+                </div>
+
+                <div class="card topics" style="width: 13rem;">
+                  <div class="card-body chicken">
+                    <img src="./images/git.svg" alt="">
+                    <h6>GIT</h6>
+                  </div>
+                </div>
+
+                <div class="card topics" style="width: 13rem;">
+                  <div class="card-body chicken">
+                    <img src="./images/databases.svg" alt="">
+                    <h6>DATABASES</h6>
+                  </div>
+                </div>
+
+                <div class="card topics" style="width: 13rem;">
+                  <div class="card-body chicken">
+                    <img src="./images/ruby.svg" alt="">
+                    <h6>RUBY</h6>
+                  </div>
+                </div>
+
+                <div class="card topics" style="width: 13rem;">
+                  <div class="card-body chicken">
+                    <img src="./images/rubyonrails.svg" alt="">
+                    <h6>RUBY ON RAILS</h6>
+                  </div>
+                </div>
+
+                <div class="card topics" style="width: 13rem;">
+                  <div class="card-body chicken">
+                    <img src="./images/gettinghired.svg" alt="">
+                    <h6>GETTING HIRED</h6>
+                  </div>
+                </div>
```

Comment:
Instead of these inline css related to width, you should improve your framework with **width** class. For example, Think about a class ```css
.width-r-13 { 
   width: 13rem 
}
```